docs/admin/admission-controllers.md

```diff
@@ -142,6 +143,15 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](../design/admission_control_limit_range.md) and the [example of Limit Range](limitrange/) for more details.
 
+### InitialResources (experimental)
+
+This plug-in observes the incoming request to create a pod and for each container without Resources specified (neither Request nor Limit)
```

Comment:
This plug-in observes pod creation requests.  For each container that does not specify compute resource requests or limits, the plug-in auto-populates a compute resource request based on historical usage of containers running the same image.